### PR TITLE
Make the TS Test AI build aircraft

### PR DESCRIPTION
**Pull request by abcdefg30**
Inspired by #18137. I noticed that our AI handles aircraft still very poorly (which might be the reason this wasn't set up before?) but fixing that is independent from the definitions, so I thought this might be a chance to sneak this in before we pick up with the AI refactor again.

### CODE REVIEW

File: mods/ts/rules/ai.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ Player:
 		PowerTypes: gapowr, napowr, naapwr
 		BarracksTypes: gapile, nahand
 		VehiclesFactoryTypes: gaweap, naweap
-		ProductionTypes: gapile, nahand, gaweap, naweap
+		ProductionTypes: gapile, nahand, gaweap, naweap, gahpad, nahand
```
Comment (Mailaender) [inline mods/ts/rules/ai.yaml]:
```suggestion
		ProductionTypes: gapile, nahand, gaweap, naweap, gahpad, nahpad
```

Comment (abcdefg30) [reply mods/ts/rules/ai.yaml]: Huh, thanks. For some reason it still built the Nod helipad and their aircraft so I didn't notice this. :eyes:

**Comment by abcdefg30** [comment]
Updated.

**Review by Mailaender — APPROVED** [review]
The AI not doing well is not an excuse for forgetting to set this up. We need a test case to develop against. The aircraft reloading is similar to Red Alert anyway.

**Review by Smittytron — APPROVED** [review]

**Pull request merged**